src/Cli/Microsoft.TemplateEngine.Cli/Microsoft.TemplateEngine.Cli.csproj

```diff
@@ -4,7 +4,6 @@
     <Description>Template creation for the dotnet CLI</Description>
     <IsPackable>true</IsPackable>
     <IsShippingPackage>false</IsShippingPackage>
-    <EnableAnalyzers>true</EnableAnalyzers>
```

Comment:
It is not extra. It enables analyzers specified in `eng/Analyzers.props` for build in development. For sourcebuild, it needs to overwrite the value when `DotNetBuildFromSource` is true in template engine projects.
```
<EnableAnalyzers>true</EnableAnalyzers>
<EnableAnalyzers Condition="'$(DotNetBuildFromSource)' == 'true'">false</EnableAnalyzers>
```